### PR TITLE
docs: Improve readme

### LINKED ISSUE #167: Improve readme file
We have several issues in our current readme file

- [x] Incorrect hierarchy for different sections
- [x] Missing license information
- [x] Project goal should be clearer
- [x] Platforms and features supported should be listed

### PR DESCRIPTION
**Pull request by orestes**
Resolves #167

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,38 @@
...
 
-A webhook to forward GitHub events to StreamLabs Alerts and Twitch Chat.
+## Goal
 
-![Apr-21-2020 9-18-24 PM](https://user-images.githubusercontent.com/7255298/79904917-bc816e00-8415-11ea-98ca-7525b169ef49.gif)
+A webhook to forward GitHub and GitLab events to StreamLabs Alerts and Twitch Chat.
 
-## Deploy to Heroku
+This is a tool created with live-coding streamers in mind. The goal is to have another source of
+interaction and showcase relevant activity on GitHub to their audience on streaming platforms
```
Comment (SantiMA10) [inline README.md:17]: or GitLab?

File: README.md [hunk 2]
```diff
@@ -9,16 +9,38 @@
...
+generating more interest in their Open Source projects.
+
+![StreamDevs Webhook Demo](https://user-images.githubusercontent.com/7255298/79904917-bc816e00-8415-11ea-98ca-7525b169ef49.gif)
+
+## Features
+
+### GitHub
```
Comment (SantiMA10) [inline README.md:24]: Should we add a `GitLab` section too?

File: README.md (hunk 1 above)
Comment (orestes) [reply README.md:17]: Moved into https://github.com/streamdevs/webhook/issues/180

File: README.md (hunk 2 above)
Comment (orestes) [reply README.md:24]: Moved into https://github.com/streamdevs/webhook/issues/180

**Pull request merged**